lib/build/loader.js

```diff
@@ -0,0 +1,82 @@
+"use strict"
+
+const path = require('path');
+const LoaderPlugin = require('./loader-plugin').LoaderPlugin;
+
+exports.generateLoaderConfig = function generateLoaderConfig(platform, bundler) {
```

Comment:
The `generate` vs. `create` naming scheme seems too ambiguous to me. What if we call them `createLoaderCode` and `createLoaderConfig`?